modules/virt-features-for-storage-matrix.adoc

```diff
@@ -88,7 +88,7 @@ endif::[]
 [.small]
 ifdef::virt-importing-vmware-vm[]
 --
-1. The `v2v-conversion-template` disk use {VirtProductName} hostpath provisioner storage if the VM disk(s) is NFS.
+1. The `v2v-conversion-template` disk must use {VirtProductName} hostpath provisioner storage if the VM disk(s) is NFS.
```

Comment:
I didn't notice this before...avoid `(s)` because it can cause translation problems. It should be fine to write, "...if the VM disks are NFS." (If not, use "...if the VM disk or disks are NFS.")

Reply:
Good catch! I hadn't thought of that. Thanks. Will fix.